src/Integration.Vsix/SonarLintDaemon/SonarLintDaemonInstaller.xaml.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.VisualStudio.Shell;
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Windows;
+
+namespace SonarLint.VisualStudio.Integration.Vsix
+{
+    /// <summary>
+    /// Interaction logic for SonarLintDaemonInstaller.xaml
+    /// </summary>
+    public partial class SonarLintDaemonInstaller : Window
+    {
+        public SonarLintDaemonInstaller()
+        {
+            InitializeComponent();
+        }
+
+        private void Window_ContentRendered(object sender, EventArgs args)
+        {
+            BackgroundWorker worker = new BackgroundWorker();
```

Comment:
This is not something that needs to be changed immediately, but I strongly recommend reading [Task.Run vs BackgroundWorker](https://blog.stephencleary.com/2013/05/taskrun-vs-backgroundworker-intro.html)